tests/components/test_system_log.py

```diff
@@ -85,6 +86,24 @@ def test_error(hass, test_client):
     assert_log(log, '', 'error message', 'ERROR')
 
 
+@asyncio.coroutine
+def test_error_posted_as_event(hass, test_client):
+    """Test that error are posted as events."""
+    events = []
+
+    @callback
+    def event_listener(event):
+        """Listen to events of type system_log_event."""
+        events.append(event)
+
+    hass.bus.async_listen(system_log.EVENT_SYSTEM_LOG, event_listener)
+
+    _LOGGER.error('error message')
+
+    assert len(events) == 1
```

Comment:
You should yield from `hass.async_block_till_done` before asserting to let the event loop run.

Reply:
Ah, but of course. Thank you! I'll fix that tomorrow.